jest.config.base.js

```diff
@@ -0,0 +1,20 @@
+const { TextDecoder, TextEncoder } = require("util");
+
+module.exports = {
+  roots: ["<rootDir>/src"],
+  transform: {
+    "^.+\\.ts$": "ts-jest",
+  },
+  moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
```

Comment:
What are the `json` and `node` file extensions used for?

Reply:
This is the default for Jest, just being explicit here. Json is used when we import json files. We could probably drop "node"